docs/estela/installation/helm-variables.md

```diff
@@ -13,7 +13,7 @@ and rename it to `values.yaml`. If you do not need to define an optional
 variable, fill its value with an empty string `""`. Now, complete the following fields:
 
 _Note_: The values that should be used if the resources have been deployed locally are
-commented in the `values.yaml.example` file.
+commented in the `values.yaml.example` file, _do not change the commented values_, you can check this [`values.yaml`]({% link estela/installation/values-yaml-example.md %}){:target="_blank"} completed file example to guide yourself.
```

Comment:
We should not give a separate `values.yaml` file as example. That's the intention of using a `.example` file.